content-cache/src/state.py

```diff
@@ -181,6 +191,8 @@ def from_integration_data(cls, data: ops.RelationDataContent) -> "LocationConfig
         backends_str = data.get(BACKENDS_FIELD_NAME, "").strip()
         fail_timeout = data.get(FAIL_TIMEOUT_FIELD_NAME, "").strip()
         backends_path = data.get(BACKENDS_PATH_FIELD_NAME, "").strip()
+        healthcheck_path = data.get(HEALTHCHECK_PATH_FIELD_NAME, "").strip()
```

Comment:
Same comment as for `content-cache-backends-config` default value

Reply:
Same answer ;)
We leave it as it is for now.